### PR TITLE
Corrects Loadout based Warnings 

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Game wise this should be no change

image

## Why It's Good For The Game

N/A - Correct me if wrong but this should have no affect in game

## Changelog
:cl:
code: Corrects loadout based warnings
/:cl:

### CODE REVIEW

**Comment by jlsnow301** [comment]
Thank you, I'll remove this from my PR, as it seemed a little out of place in mine https://github.com/BeeStation/BeeStation-Hornet/pull/1699

**Comment by ike709** [comment]
I don't have time to investigate right now but this may have unintended consequences involving `populate_gear_list()`

**Comment by Trilbyspaceclone** [comment]
@ike709 
Right the issue as it is, 
`I am gear/tie, ok what is my display name? Right none set, let me look at my item, no item, ok let me look at my default setting, nothing. I dont know what to name myself`
Its doing that cuz all three are branching without-names of things and theirs a check to see what their names are. It looks at the var and goes "" < --- That is really bad for the complier to handle as its trying twice to place something inside but fines not null but flat out nothing.

**Comment by Trilbyspaceclone** [comment]
@ike709 Any reason for not reviewing this pr?

**Comment by ike709** [comment]
Finals week.

File: code/modules/client/loadout/loadout_accessories.dm [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /datum/gear/accessory
+	display_name = "accessory" //So we dont give warnings
```
Comment (ike709) [inline code/modules/client/loadout/loadout_accessories.dm:2]: Instead you should create a `/datum/gear/accessory` and set the `subtype_path`.

File: code/modules/client/loadout/loadout_ooc.dm [hunk 2]
```diff
@@ -1,6 +1,9 @@
+/datum/gear/ooc
+	display_name = "OOC Loadout Gear" //So we dont give warnings
+	sort_category = "OOC"
```
Comment (ike709) [inline code/modules/client/loadout/loadout_ooc.dm:3]:
```suggestion
	sort_category = "OOC"
	subtype_path = /datum/gear/ooc

```

**Comment by ike709** [comment]
#1575 fixes these issues

**Pull request closed without merging**